src/main/java/com/reimbursements/reporitories/AuthorityRepository.java

```diff
@@ -0,0 +1,17 @@
+package com.reimbursements.reporitories;
+
+
+
+import java.util.Set;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+
+import com.reimbursements.models.Authority;
+
+
+@Repository
+public interface AuthorityRepository extends JpaRepository<Authority, Integer> {
+	public Set<Authority> getByAuthorityID(int authorityID);

```

Comment:
This method is not required. By default, `JpaRepository` implements some basic methods: get by ID is one of them.